### DISCUSSION TITLE
領錢會不會留明細

### DISCUSSION THREAD

**Discussion by Aaron0810** [1101-GS4719-C/Python-Community-GS4719-C#1461] [General]
前幾天跟朋友在郵局領錢，他看到我領完錢拿著交易明細走出去，就問說我會把那個印出來喔? 我就說會阿，這樣也比較清楚自己領錢的頻率也可以順便記帳，才不會短時間不小心花太多錢，但他又跟我說他覺得領錢還印那個明印很浪費紙、不環保，我後來想想好像也還蠻有道理的，因為我是那種能用自己的餐具就不會去拿免洗筷的那種，害我現在領錢都不知道要不要把明細印出來了。

不知道大家平常領錢是會不會印明細的，順便給我個建議不然現在自己有點矛盾。

**Comment by ALAN30359704** [comment]
我本人是不太會留明細，因為現在手機可以直接下載網銀的App，所以如果我要確定明細的話只要開手機的網銀就可以了！

**Comment by alstinlin** [comment]
我通常不會將明細印出來，一來能減少紙張的浪費，二來能降低資料外洩的風險，所以我都會在提款後直接在螢幕上確認目前的餘額，就不需要再額外把明細列印出來了

**Comment by alicia0725** [comment]
我認為將明細印出來會有點多次一舉，如果沒收好可能會造成帳號洩漏等問題，並且現在網頁銀行已經可以完美替代這個問題了

**Comment by ashleyy-huang** [comment]
通常把明細印出來不只會需要多攜帶一張廢紙，也相當不環保，通常交易紀錄和出入帳等都可以透過載具和網路銀行來記錄，我也可以隨時透過手機查詢，不用一直保存著明細，這樣方便許多了

**Comment by aa103967622** [comment]
現在手機軟體裡面都有紀錄明細資料的功能了！像是郵局的app，只要你登記自己的金融卡在上面，他就會紀錄每一次你不論用手機刷條碼付費，或是去atm領錢，app上面都會有詳細的紀錄，所以我認為是不需要領取紙本明細的，就算拿了其實回去也是馬上丟掉。

**Comment by pigeon9975** [comment]
如果只是單純的提款我不會留名細耶，感覺看一眼就丟掉一張紙好浪費，但如果是匯款給別人，尤其是那種比較大的款項(民宿訂金之類的)，我就會把明細留著，以免別人收不到錢找我討債然後我沒有證據堵住他的嘴。

**Comment by poweichen00** [comment]
我不會留明細，有網銀app一看就知道，有些也可以綁line，也會直接傳資訊給你。

**Comment by youtinglin** [comment]
我不會印明細，因為也是認為不環保，而且現在很方便，看網銀帳戶就可以知道自己的收入和支出的交易明細，如果沒有網銀帳戶，也可以用記帳的app，看自己花錢的頻率。

**Comment by zephyrxin01** [comment]
我覺得就看自己的需求吧，如果你認為明細比較方便你核查自己的帳目，就列印吧。因為也不是所有人都有網銀，我就是郵局網銀的登入資訊忘記但又沒有時間去現場改資料的一員。但我通常不會印下來，習慣自己記帳，因為我覺得網銀紀錄的交易資料很容易看不出來消費了什麼東西(公司行號很難分辨，日期也不見得是消費當日的資訊)

**Comment by Alexgns** [comment]
目前大家應該都有網路銀行了，因此明細相較以前已經沒那麼重要了，想看帳目隨時都可看，所以我都沒有在印明

**Comment by pudding0803** [comment]
我一開始也是都會印出來，但後來覺得不太環保，而且其實印完通常只會看一次(?)，所以我現在都是直接看螢幕上顯示的，另外我在手機也可以隨時看到存款，因此我覺得印出來似乎沒有太大的意義嗎🤔

**Comment by Tzuhsuannnnn** [comment]
我不會將明細印出來，一是怕個資洩漏，老一輩的人好像都會習慣印出來後撕掉，但我常常拿了就不知道放去哪，也會忘記要撕掉丟掉，所以有一點危險。二是環保，現在連發票都流行電子發票，我認為明細可以少印一張就少印一張。

**Comment by sumairu0727** [comment]
我是有在手機下載網路郵局，這樣更方便記帳或即時查看餘額，也時候忘記近期花了多少錢打開手機看一下就知道了

**Comment by Superlan8** [comment]
我也是不會印明細的人，除了繳費需要拍明細給對方，不然我覺得印明細有點浪費地球資源，有時候印出來又會忘記在哪裡，還不然下載網路郵局，忘記餘額的話上網看一下便知，節省時間又節省地球資源。

**Comment by chia-kai-chang** [comment]
大會報告，閒聊八卦且沒有科技成分的discussion，無法得到點數。

**Comment by Brownnnn78** [comment]
現在手機支付真的超方便，街口支付和line pay也超級方便，每天消費都能累積一元一元的回饋，讓我最近很少領錢順便能一點一點省錢，且手機載具綁定記帳軟體也可以自動幫忙記帳。推薦給大家這些生活結合手機應用的小撇步。
